huff-language/huffc
# feat: add ERC20 metadata

This PR adds Metadata (Name, Symbol) to the ERC-20 example by using the code done by @exp-table at [PR #25](https://github.com/huff-language/huffc/pull/25)

## Test Refractor
+ Use `[] = await ethers.getSigners();` to get the signers
+ Make use of `signer.address` instead of new variables
+ Added a metadata check test
